backend/core/src/auth/services/user.service.ts

```diff
@@ -562,12 +567,27 @@ export class UserService {
   }
 
   private async authorizeUserAction(requestingUser, targetUser, action) {
+    if (requestingUser.roles?.isJurisdictionalAdmin) {
+      return this.authorizeJurisdictionalAdmin(requestingUser, targetUser)
+    }
     return await this.authzService.canOrThrow(requestingUser, "user", action, {
       id: targetUser.id,
       jurisdictionId: targetUser.id,
     })
   }
 
+  private authorizeJurisdictionalAdmin(requestingUser, targetUser) {
+    // jurisdictional admins can't view super admins
+    if (targetUser.roles?.isAdmin) {
+      throw new HttpException("Forbidden", HttpStatus.FORBIDDEN)
+    }
+
+    const requesterJurisdictions = requestingUser.jurisdictions?.map((juris) => juris.id)
+    const targetJurisdictions = targetUser.jurisdictions?.map((juris) => juris.id)
+    // jurisdictional admins should only see a user if they share a jurisdiction
+    return requesterJurisdictions.some((juris) => targetJurisdictions.includes(juris))
+  }
```

Comment:
What do you think about leaving authorizeUserAction as it was and updating the use `list` service to add a filter to the query if the user is a jurisdictional admin to get users who are not isAdmin and to only get users who are in the same jurisdiction? And maybe that would be a separate query to get the IDs first and then query by ids.